ext/http/compressible.rs

```diff
@@ -0,0 +1,660 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::ByteString;
+
+// Data obtained from https://github.com/jshttp/mime-db/blob/fa5e4ef3cc8907ec3c5ec5b85af0c63d7059a5cd/db.json
+// Important! Keep this list sorted alphabetically.
+const CONTENT_TYPES: &[&str] = &[
```

Comment:
Nit: there are some new additions to https://github.com/jshttp/mime-db/blob/master/db.json; we might want to update the list.

feel free to skip it

Reply:
Let's skip it for now, as they are very likely very esoteric media types that aren't widely used.

Longer term, someone should either find an efficient way to statically include the JSON in, or generate the code statically from the JSON file.